### DISCUSSION TITLE
体力回复问题

### DISCUSSION THREAD

**Discussion by wangshun111** [NateScarlet/auto-derby#191] [Q&A] [answered]
在体力很少的时候会倾向于打比赛或者外出提升心情，导致体力状况很差，影响后期训练，个人希望能增加回体动作的权重，需要怎么做？大哥之后会对这个问题更新吗？

**Comment by NateScarlet** [comment, marked as answer]
这不算问题 只是训练策略不同 所有自定义策略都用插件解决

通用方法是参考 [plugins/example_custom_training_score.py](https://github.com/NateScarlet/auto-derby/blob/e9bdca21a6dc82460b3a27941943687d52b2d651/plugins/example_custom_training_score.py) 插件覆盖训练评分

给回体力更多分数，影响休息/外出/友人卡/智力训练回体

仅限休息评分就覆盖 config.single_mode_rest_score 进行额外加分

https://github.com/NateScarlet/auto-derby/blob/e9bdca21a6dc82460b3a27941943687d52b2d651/auto_derby/_config.py#L80

**Reply by NateScarlet** [reply]
如果你觉得是问题应该找出一回合明显应该选休息结果没选的日志

所有行动的评分都在日志里
